terraform/modules/bootstrap_concourse/sg_bootstrap.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_security_group" "bootstrap" {
+  ingress {
+    from_port = 22
+    to_port = 22
```

Comment:
I think this is definitely a YAGNI. The only reason you'd need it is if Concourse itself doesn't come up, but then that points to a problem with the AMI, which means digging into the packer setup.